### ISSUE TITLE
accessibilityLabel missing

### ISSUE THREAD

**Issue by josenaves** [ide/react-native-button#68]
I'm using this component in our app but have found that it does not uses accessibilityLabel.
I need to have accessibilityLabel set to my Appium end-to-end tests to work.

**Comment by josenaves** [comment]
Hey @ide. I'm going to close this issue as #69 is merged.
Just one question, are you going to make a official release on npm ?

**Comment by ide** [comment]
@josenaves hi, I published 2.2.0 to npm.

**Comment by josenaves** [comment]
Thank you for let me know @ide !
